src/targets/happymodel_2400.ini

```diff
@@ -0,0 +1,35 @@
+
+# ********************************
+# Transmitter targets
+# ********************************
+
+[env:HappyModel_ES24TX_2400_TX_via_UART]
+extends = env:DIY_2400_TX_ESP32_SX1280_E28_via_UART
+
+[env:HappyModel_ES24TX_2400_TX_via_WIFI]
+extends = env:HappyModel_ES24TX_2400_TX_via_UART
+
```

Comment:
Do you need an `upload_port` here similar to the WIFI below?

Reply:
Upload port is defaulted in python script so It is not needed here :)

Reply:
For WIFI to 10.0.0.1?

Reply:
https://github.com/ExpressLRS/ExpressLRS/blob/2568f125a8f72d5776efec62da6ba162bfd036c8/src/python/upload_via_esp8266_backpack.py#L7
It uses dns because WifiManager can be used to connect backpacklogger to home LAN. This same need to be implemented to RXes as well but atm RX uses AP mode only with fixed IP.

Reply:
That all makes sense, except `build_env_setup.py` only seems to use `upload_via_esp8266_backpack` if it's `stm` with some other options, or `espressif8266` at line 40.

If it's supposed to set the uploader to that for `espressif32` platforms because they can have a backpack then that needs to be fixed too. i.e. added to list in `build_env_setup.py` at line 40.

Theres also a web-update directly on the ESP32 which sets up an AP with SSID `ExpressLRS TX Module` with IP address 10.0.0.1, which allows updates.

So, theres two ways of doing a wifi update on the ESP32 based TX modules, one via the ESP32 WIFI and the other via the backpack. So i'm not sure what method we should be using for this module, it's got so many options! 😄

Reply:
We can play with the options here later in another PR. This one will be needed as soon as people start getting their hardware, so there is some sense of urgency.

Reply:
Oh yeah, that is ESP32 😬
Sorry @pkendall64